main.go

```diff
@@ -86,15 +85,16 @@ func main() {
 
 	nnfopts := nnf.BindFlags(flag.CommandLine)
 
-	opts := zapcr.Options{
+	opts := &zap.Options{
 		Development: true,
+		TimeEncoder: zapcore.ISO8601TimeEncoder,
```

Comment:
Do you expect this to make all logs report in the same format?

Reply:
No. It's just a means to default the zap time encoder to ISO8601 instead of hardcoding it.